mk/board.mk

```diff
@@ -0,0 +1,3 @@
+ifeq ($(BOARD),de10nano)
+    CFLAGS += -mcpu=cortex-a9 -mfloat-abi=softfp -mfpu=neon
```

Comment:
Add `-mtune=cortex-a9`
Reference: https://gist.github.com/fm4dd/c663217935dc17f0fc73c9c81b0aa845

Reply:
By the way, we can might try `-mneon-for-64bits` option for de10-nano board.

Reply:
I'll add `-mtune=cortex-a9`.
However, I'm not sure about the necessity of the option `-mneon-for-64-bit`.
Based on the GCC manual and the reference, it says: **It is disabled by default due to high cost of moving data from core registers to Neon on 32bit CPU's.**
Out Cortex-A9 CPU is 32-bit.

Reply:
I have added `-mtune=cortex-a9` and does not include the `-mneon-for-64bits`.